packages/widgetbook/lib/src/routing/story_router_delegate.dart

```diff
@@ -77,7 +79,18 @@ class StoryRouterDelegate<CustomTheme> extends RouterDelegate<StoryRoutePath>
                       width: 16,
                     ),
                     Expanded(
-                      child: Workbench<CustomTheme>(),
+                      child: Row(
+                        children: [
+                          Expanded(
+                            flex: 3,
+                            child: Workbench<CustomTheme>(),
+                          ),
+                          const SizedBox(width: 16,),
+                          const Expanded(
+                            child: SettingsPanel(),
+                          )
+                        ],
```

Comment:
I like the usage of flex here (in comparison to the navigation panel which is hard coded to a specific value). However, this can be problematic for smaller screens since the text within the Panel might consume a lot of vertical space. 

See the usage of Widgetbook for a screen which is only 1440 pixels wide: 

![image](https://user-images.githubusercontent.com/16862002/155461133-c19a0074-deb2-487b-a1d0-c0fc71a5d7df.png)

Reply:
But small screens are a problem in general 😀

Reply:
> small screens are a problem in general

facts haha